frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer.tsx

```diff
@@ -20,6 +20,7 @@ import Collections, {
 } from "metabase/entities/collections";
 import { openNavbar, closeNavbar } from "metabase/redux/app";
 import { logout } from "metabase/auth/actions";
+import { canAccessAdmin } from "metabase/admin/app/selectors";
```

Comment:
Given the upcoming changes in permissions, I suppose we can have users with access to the admin site but without database management permissions. @alxnddr is that correct? If so, I believe we need to use another selector

Reply:
@gusaiani @kulyk @alxnddr — any resolution here?

Reply:
Sorry, I missed the notification. I assume we need to show this button only to users who actually can add databases. This did not change and only admins can add databases. So please don't use `canAccessAdmin` here because it will return true for non-admins who have certain permissions

Reply:
@alxnddr thanks, I've changed to selector to `isUserAdmin`. Is this 👍?